src/config.json

```diff
@@ -0,0 +1,72 @@
+{
+    "GUILD": {
+        "ID": 1368604371323064421,
+        "ROLES": {
+            "admin": 1387103854381760553,
+            "mod": 1387103950099972268,
+            "botDev": 1386724072074772610,
+            "linked": 1386815940791894198,
+            "just_joined": 1389256789513470042,
+            "muted": 1386947661084495913,
+            "BRANCH":{
+                "CSE": "984846616580198450",
+                "CSE (AI&ML)": "1061350128939716768",
+                "ECE": "984846618371174400",
+                "EEE": "984846620157964389",
+                "ME": "984846621848248320",
+                "BT": "984846623676969030",
+                "CV": "984846625446985728",
+                "B ARCH": "984846627158257684",
+                "BBA": "984846628596899881",
+                "B.DES": "984846630396235850",
+                "BBA LLB": "984846632405315646",
+                "BBA-HEM": "984846634385047632",
+                "BA LLB": "984846636226314270",
+                "BBA - Sports Management": "984846638025670726",
+                "BCA": "984846639841820752",
+                "B.Com": "984846642354192484",
+                "BBA (Hons) in Business Analytics": "1023509952964341820",
+                "B.Com (Hons) with ACCA": "1023510367026036826",
+                "Psychology": "1023510685705044009",
+                "Sports Management": "1023511154649223240",
+                "Bachelor of Pharmacy": "1023512100724817940",
+                "Nursing": "1061350434675101726",
+                "CA": "1086905421291335711",
+                "International Accounting and Finance": "1129414449485316176",
+                "Business Analytics": "1136371141330608138",
+                "MBA": "1289303483522093127",
+                "MBBS": "1336785790730108978"
+            },
+            "YEAR": {
+                "2015": "1119203107130318889",
+                "2016": "1106834902667759717",
+                "2017": "1079825096287453244",
+                "2018": "984846644031942697",
+                "2019": "984846646271696977",
+                "2020": "984846648112971867",
+                "2021": "984846649488732161",
+                "2022": "1023513091994025994",
+                "2023": "1123313984163041410",
+                "2024": "1244601965514588170",
+                "2025": "1381083961895157813"
+            },
+            "CAMPUS": {
+                "RR": "984872936529887322",
+                "EC": "984873178339897384"
+            }
+        },
+        "CHANNELS": {
+            "LOGS": {
+                "logs": 1393983999113822399,
+                "modlogs": 1386770381733630012,
+                "nqnlogs": 1368785403020316692
```

Comment:
this is not in the same consistent format as the channels below (and [this](https://github.com/pesu-dev/auth-link-portal/blob/app-router/src/utils/config.js#L6))